packages/edit-site/src/components/sidebar-edit-mode/template-panel/template-actions.js

```diff
@@ -11,13 +11,25 @@ import { moreVertical } from '@wordpress/icons';
  */
 import { store as editSiteStore } from '../../../store';
 import isTemplateRevertable from '../../../utils/is-template-revertable';
-
+import ReplaceTemplateButton from './replace-template-button';
+import { useAvailablePatterns } from '../page-panels/hooks';
 export default function Actions( { template } ) {
+	const availablePatterns = useAvailablePatterns( template );
+	const availableTemplates = availablePatterns.map( ( pattern ) => {
+		return {
+			name: pattern.name,
+			blocks: pattern.blocks,
+			title: pattern.title,
+			content: pattern.content,
+		};
+	} );
```

Comment:
Nit: Can we make this part of the `useAvailablePatterns` or `preparePatterns`?

Reply:
I think we can just remove it :)